src/CMakeLists.txt

```diff
@@ -293,6 +293,9 @@ if(APPLE)
         target_link_libraries(keepassx_core "-framework LocalAuthentication")
     endif()
 endif()
+if(HAIKU)
+    target_link_libraries(keepassx_core network)
```

Comment:
What is this library?

Reply:
libnetwork.so is where network fictions live in Haiku.

Reply:
would you only want this if "WITH_XC_NETWORKING" is true?